### ISSUE TITLE
Support timestamps without ms

### ISSUE THREAD

**Issue by heumat** [tkarabela/pysubs2#50]
Hey,

I had issues when reading `.ass` files where the timestamps do not have a millisecond attached. I suggest / propose to add this support as it can not be expected that all timestamps always have this information.

The fix is quite easy, but I can not push a branch to the repo, so I include it here:

- file `time.py`:
  ```
  TIMESTAMP = re.compile(r"(\d{1,2}):(\d{2}):(\d{2})[.,]?(\d{0,3})")
  ```

  in function `timestamp_to_ms`:
    ```
        if groups[-1] == '':
            h, m, s = map(int, groups[:-1])
            ms = 0
        else:
            h, m, s, frac = map(int, groups)
            ms = frac * 10**(3 - len(groups[-1]))
    ```

- The corresponding tests, `test_time.py` need a small update too:
  ```
      assert TIMESTAMP.match("12:45:67").groups() == ("12", "45", "67", '')
      assert TIMESTAMP.match("1:23:45,").groups() == ("1", "23", "45", '')
      assert TIMESTAMP.match("1:23:45.").groups() == ("1", "23", "45", '')
  ```
  and remove them from the rejected timestamps section.

I hope the changes make sense to you!

**Comment by heumat** [comment]
Hey @tkarabela , of course :) I ran the tests and adapted what broke so I hope that also covers some aspects. You can let me know if you want some assistance with the checks.
Thanks!

**Comment by tkarabela** [comment]
@heumat just letting you know that I released pysubs2 version 1.4.2 where H:MM:SS timestamps in ASS are supported :)

**Comment by heumat** [comment]
@tkarabela , that's amazing! Thank you very much :)
